tests/test_external_strategies.py

```diff
@@ -0,0 +1,125 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from unittest.mock import MagicMock, Mock
+
+from d3a.models.area import Area
+from d3a.models.strategy.external_strategies.load import LoadForecastExternalStrategy
+from d3a.models.strategy.external_strategies.pv import PVForecastExternalStrategy
+from d3a_interface.constants_limits import ConstSettings
+from d3a_interface.constants_limits import GlobalConfig
+from pendulum import now, duration
+from pytest import mark
+
+
+def ext_strategy_fixture(strategy):
+    config = Mock()
+    config.slot_length = duration(minutes=15)
+    config.tick_length = duration(seconds=15)
+    config.ticks_per_slot = 90
```

Comment:
correct me if I am wrong, it should be 60

Reply:
We only need a number here and independent of the value. I will change it anyway as you are right.